Release notes for 0.25.5 (#414)
astronomer/docs: enterprise/v0.25/10_resources/01_release-notes.md

We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).

## v0.25.5

Release Date: July 29, 2021

### Ability to Hard Delete Airflow Deployments

> Astronomer v0.25.5 introduces a platform-level setting that enables the ability for Deployment Admins to hard delete Airflow Deployments from Astronomer. For more information about this feature, read [Delete a Deployment](https://www.astronomer.io/docs/enterprise/v0.25/deploy/configure-deployment#delete-a-deployment).

Reviewer: ```suggestion
Astronomer v0.25.5 introduces a platform-level setting that gives Deployment Admins the ability to hard delete Airflow Deployments. For more information about this feature, read [Delete a Deployment](https://www.astronomer.io/docs/enterprise/v0.25/deploy/configure-deployment#delete-a-deployment).
```

### Airgapped Installation

> For customers interested in running Astronomer in an airgapped environment with no access to the internet, we've added the ability to do so. To learn more, contact [Astronomer Support](https://support.astronomer.io).

Reviewer: ```suggestion
Organizations can now install Astronomer in an airgapped environment with no access to the internet. To learn more, contact [Astronomer Support](https://support.astronomer.io).
```

We're committed to testing all Astronomer Enterprise versions for scale, reliability and security on Amazon EKS, Google GKE and Azure AKS. If you have any questions or an issue to report, don't hesitate to [reach out to us](https://support.astronomer.io).

## v0.25.5

Release Date: July 29, 2021

### Ability to Hard Delete Airflow Deployments

> Astronomer v0.25.5 introduces a platform-level setting that enables the ability for Deployment Admins to hard delete Airflow Deployments from Astronomer. Once that setting is enabled, users can choose to hard delete a Deployment in the **Settings** tab of the Astronomer UI or by running `$ astro deployment delete --hard` via the Astronomer CLI.

Reviewer: ```suggestion
Astronomer v0.25.5 introduces the ability for Deployment Admins to hard delete Airflow Deployments from Astronomer. This feature must first be enabled in your platform configuration before users can access it. 

Once the feature is enabled, users can hard delete a Deployment in the **Settings** tab of the Astronomer UI or by running `astro deployment delete --hard` via the Astronomer CLI.
```

This action permanently deletes all data associated with a Deployment that cannot be recovered, including the database and underlying Kubernetes resources. If your team has enabled [custom release names](https://www.astronomer.io/docs/enterprise/v0.25/deploy/configure-deployment#customize-release-names), hard deleting a Deployment unlocks its release name for reuse by another Deployment.

> 

Reviewer: ```suggestion

To enable this feature, set `deployments.hardDeleteDeployment = true` in your `config.yaml` file and push the changes to your platform as described in [Apply a Config Change](https://www.astronomer.io/docs/enterprise/v0.25/manage-astronomer/apply-platform-config).

```

### Airgapped Installation

> For customers interested in running Astronomer in an airgapped environment with no access to the internet, we've added the ability to do so. To learn more, contact [Astronomer Support](https://support.astronomer.io).

Reviewer: ```suggestion
Organizations can now install Astronomer in an airgapped environment with no access to the internet. To learn more about this installation process, contact [Astronomer Support](https://support.astronomer.io).
```
